integreat_cms/static/src/js/analytics/hix-widget.ts

```diff
@@ -190,7 +190,12 @@ window.addEventListener("load", async () => {
             setHixLabelState("no-content");
             document.getElementById("hix-loading")?.classList.add("hidden");
         } else {
-            const initialHixValue = await getHixValue(updateButton.dataset.url);
+            let initialHixValue = parseInt(document.getElementById("hix-container").dataset.initialHixScore, 10);
+            if (initialHixValue) {
+                setHixLabelState("updated");
```

Comment:
Update: this problem occrues only when HIX is enabled but DeepL not (because some `Error: Cannot read properties of null` will be triggered). We can dismiss this if it is assumed that HIX is enabled only when DeepL is enabled.

Reply:
> We can dismiss this if it is assumed that HIX is enabled only when DeepL is enabled.

I think we cannot always assume this, so we should support these features independently.

Reply:
I would even consider it a bug that the loading animation does no longer work when DeepL is enabled :thinking:

Reply:
For reference: @JoeyStk now opened an issue for the broken loading animation: https://github.com/digitalfabrik/integreat-cms/issues/2222

Reply:
Am I correct in assuming that this suggestion is out of date / can safely be ignored?

Reply:
No, the problem still exists: When HIX is enabled, but DeepL is not, and the cached value is used, the loading animation just spins forever:

![Screenshot 2023-04-12 at 11-02-53 Integreat Editorial System](https://user-images.githubusercontent.com/16021269/231408554-154ce7b2-e428-4438-8331-d876bb5993bb.png)

The only reason this is limited to the case "DeepL is disabled" is the bug referenced in the new ticket (after the bug is fixed, the problem with the infinite loading spinner would occur independently of whether DeepL is enabled or not).